Firebase/Auth/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # v5.4.1
 - Deprecate Microsoft and Yahoo OAuth Provider ID (#2517)
 - Fix an issue where an exception was thrown when linking OAuth credentials. (#2521)
-- Fix an issue where a wrong error was thrown when upgrading anonymous user. (#2530)
+- Fix an issue where a wrong error was thrown when handling error with
+  FEDERATED_USER_ID_ALREADY_LINKED. (#2530, #2542)
```

Comment:
Note that the general preference is to use Issue numbers over PR numbers when both apply.